services/galley/package.yaml

```diff
@@ -98,6 +98,7 @@ executables:
     ghc-options:
     - -threaded
     - -with-rtsopts=-T
+    - -rtsopts
```

Comment:
What does this flag do again? (is it related to this PR?)

Reply:
`-with-rtsopts` passes implicit options to the run time system every time the executable we build here is started.  `-rtsopts` makes the run time system capable of understanding these arguments.  I somehow hope that the latter is implicit in the former.

(off the top of my head, i haven't checked the docs.)

Reply:
Thanks!

Reply:
`rtsopts` allows you to override RTS opts at runtime; otherwise they are ignored. Not *directly* related to the other stuff going on; but is necessary for running certain types of profiling without recompiling.